content/faq/lbry-privacy.md

```diff
@@ -0,0 +1,27 @@
+---
+title: How does LBRY handle Privacy, and what data do you collect??
+category: other
+---
+## Is it possible to publish content anonymously? How is anonymity accomplished?
+
+LBRY does not log user download activity. However, by using our software, you are not anonymous on the network by default, similar to the way most torrenting software works. There are ways you can be more anonymous by using a VPN or other technologies. Please also refer to our [FAQs on Acceptable Use Policy](https://lbry.io/faq/acceptable-use-policy) if you’re thinking about doing anything illegal, as we don’t tolerate that.
```

Comment:
upload activity? We do log downloads for rewards purposes, and it sounds like we may do the same for uploads at some point down the road.